File: check_product_api.js
const fetch = require('node-fetch'); // You might need to install node-fetch or use built-in fetch if node version > 18

async function checkProduct() {
    try {
        const response = await fetch('http://localhost:5000/api/products/20?_t=' + Date.now());
        const data = await response.json();
        console.log('Product Image URLs:', data.imageUrls);
        console.log('Product Name:', data.name);
    } catch (error) {
        console.error('Error:', error);
    }
}

checkProduct();

File: check_product_api.py
import requests
import json

try:
    response = requests.get('http://localhost:5000/api/products/20')
    if response.status_code == 200:
        data = response.json()
        print("Image URLs served by API:")
        print(json.dumps(data.get('imageUrls'), indent=2))
    else:
        print(f"Error: {response.status_code}")
except Exception as e:
    print(f"Failed to connect: {e}")
